lib/database/calender.ts

```diff
@@ -0,0 +1,25 @@
+import { client } from "../db";
+import { Any } from "io-ts";
+
+// Creates a calender event in database
+const createCalenderEvent = async (
+  id: string,
+  startString: string,
+  endString: string
+): Promise<Any[]> => {
```

Comment:
I'm assuming this Any[] is because an empty array is returned? Maybe add a comment like the one in lib/database/commitments.ts so it's clear